lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,22 @@
+desc 'refresh will rebase your current branch based on the base branch'
+long_desc <<LONGTIME
+  Performs the following:\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull --rebase <remote_location> <base_branch>\n
+  \t$ git checkout <current_branch>\n
+  \t$ git rebase <base_branch>\n
+LONGTIME
+arg_name '[base_branch] - branch that you want to rebase on (master by default), [remote_location] - remote repository name to fetch updates from (origin by default)'
+command :refresh do |c|
+  c.action do |global_options, options, args|
+    if args.empty?
+      raise "usage: git reflow refresh [base_branch_name - defaults to master] [remote_location - defaults to origin]"
+    else
+      # defaults remote_location to origin
+      remote_location = args.length < 2 ? "origin" : args[1]
+      base_branch = args.length < 1 ? "master" : args[0]
+      
+      GitReflow.update_destination remote_location base_branch
```

Comment:
All Rspec tests are currently passing. Can replace lines 16-19 with a 1 line ternary operation, but it is a bit long and you might tell me about code readability 👍